packages/slice-machine/src/legacy/components/AppLayout/index.tsx

```diff
@@ -18,7 +26,31 @@ export const AppLayout: FC<PropsWithChildren> = ({
   ...otherProps
 }) => {
   return (
-    <ErrorBoundary>
+    <ErrorBoundary
+      renderError={(error) => {
+        return (
+          <Box
+            position="absolute"
+            top={64}
+            width="100%"
+            justifyContent="center"
+            flexDirection="column"
+          >
+            <BlankSlate>
+              <BlankSlateIcon
+                lineColor="tomato11"
+                backgroundColor="tomato3"
+                name="alert"
+              />
+              <BlankSlateTitle>Failed to load Slice Machine</BlankSlateTitle>
+              <BlankSlateDescription>
+                {JSON.stringify(error)}
```

Comment:
isn't it better to show `error.message` or something like that?

Reply:
As it's the global main ErrorBoundary the goal is really for us (Prismic) to have everything possible to debug and understand user issues. error name is also super important and any other format type of error would be displayed thanks to that.